orchagent/port/porthlpr.cpp

```diff
@@ -118,6 +118,14 @@ static const std::unordered_map<std::string, Port::Role> portRoleMap =
     { PORT_ROLE_DPC, Port::Role::Dpc }
 };
 
+static const std::unordered_map<std::string, sai_port_loopback_mode_t> portLoopbackModeMap = {
+    {"none",      SAI_PORT_LOOPBACK_MODE_NONE},
+    {"phy_local", SAI_PORT_LOOPBACK_MODE_PHY},
+    {"mac_local", SAI_PORT_LOOPBACK_MODE_MAC},
```

Comment:
@royyi8 how is it different from `SAI_PORT_INTERNAL_LOOPBACK_MODE_MAC`

Reply:
`SAI_PORT_LOOPBACK_MODE_MAC` is the corresponding attribute for `SAI_PORT_INTERNAL_LOOPBACK_MODE_MAC`. We are using `sai_port_loopback_mode_t` enum instead of `_sai_port_internal_loopback_mode_t` since the later is to be deprecated in SAI.

https://github.com/opencomputeproject/SAI/blob/master/inc/saiport.h#L114